lib/msal-angular/src/msal.broadcast.service.ts

```diff
@@ -22,8 +22,11 @@ export class MsalBroadcastService {
     ) {
         this._msalSubject = new Subject<EventMessage>();
         this.msalSubject$  = this._msalSubject.asObservable();
-        this._inProgress = new Subject<InteractionStatus>();
+
+        // InProgress as BehaviorSubject so most recent inProgress state will be available upon subscription
+        this._inProgress = new BehaviorSubject<InteractionStatus>(undefined);
```

Comment:
Unsure whether to set it to undefined, null, or `Interaction.None`. Thoughts?

Reply:
When is the default value used? In react we always start with `InteractionStatus.Startup` knowing that we will always call `handleRedirectPromise` at some point which will set it to `None` when complete. You likely don't want it starting at `None` as that can provide a brief window when that would be inaccurate (i.e. handleRedirectPromise needs to resolve an in progress login)

Reply:
Startup seems like a good first value.

Reply:
The default value hasn't really been used, and only with changing it to a `BehaviorSubject` was an initial value available. While  `handleRedirectObservable` should always be called and set it to `None`, I agree that setting it to `Startup` would be more accurate.